code/modules/jobs/job_types/bighorn.dm

```diff
@@ -434,8 +434,8 @@ Mayor
 	/datum/outfit/loadout/cleanser		//Just some bombs.
 	)
 
-	access = list()		//we can expand on this and make alterations as people suggest different loadouts
-	minimal_access = list()
+	access = list(ACCESS_BAR)		//we can expand on this and make alterations as people suggest different loadouts
```

Comment:
Preacher becoming an alcoholic, yo????

Reply:
The Bighorn Bar's access is Kitchen.
The town's own access, for like the workshop/ect, is Bar.

Reply:
I.. Honestly don't know why that is, but I figure actual town access would be something to fix for BH2.0